web/packages/core/src/public/player/legacy.ts

```diff
@@ -1,56 +1,106 @@
 import { DataLoadOptions, URLLoadOptions } from "../config";
 import { MovieMetadata } from "./movie-metadata";
-import { ReadyState } from "../../internal/player/inner";
+
+/**
+ * Describes the loading state of an SWF movie.
+ *
+ * @deprecated Use {@link PlayerV1.readyState} to get a {@link ReadyState} instead.
+ */
+export enum LegacyReadyState {
```

Comment:
Can we remove this enum and just compare any legacy ready states to numbers instead of enum members? This only adds confusion.

Reply:
I'm not sure that having people just need to know what arbitrary numbers means is better than code constants (which an enum basically is). That seems even more confusing to me.
Adrian argued in favour of just keeping the legacy enum and not having the new one, which whilst I'm not personally in favour of, I think I'd prefer over switching and entirely not documenting the old style :(

Reply:
Reverted to old style API, and made the old enum public